src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -564,43 +557,28 @@ namespace CommandLineInterface {
         }
 
         // Check if specified files exist
-        {
-            IOFlags flags;
-            ObjexxFCL::gio::inquire(inputFileName, flags);
-            FileExists = flags.exists();
-        }
-        if (!FileExists) {
+        if (!fileExists(inputFileName)) {
             DisplayString("ERROR: Could not find input data file: " + getAbsolutePath(inputFileName) + ".");
             DisplayString(errorFollowUp);
             exit(EXIT_FAILURE);
         }
 
         if (opt.isSet("-w") && !DDOnlySimulation) {
-            {
-                IOFlags flags;
-                ObjexxFCL::gio::inquire(inputWeatherFileName, flags);
-                FileExists = flags.exists();
-            }
-            if (!FileExists) {
+            if (!fileExists(inputWeatherFileName)) {
                 DisplayString("ERROR: Could not find weather file: " + getAbsolutePath(inputWeatherFileName) + ".");
                 DisplayString(errorFollowUp);
                 exit(EXIT_FAILURE);
             }
         }
 
-        OutputFiles::getSingleton().debug.ensure_open();
+        OutputFiles::getSingleton().debug.ensure_open("OpenOutputFiles");
```

Comment:
Can you clarify this action?  It looks like it is making sure the "OpenOutputFiles" file is open.

Reply:
It's opening the `debug` file and using the string `"OpenOutputFiles"` in the error report if something goes wrong - it's the name of the calling function.

Reply:
Thanks for the clarification